src/site/markdown/index.md

```diff
@@ -0,0 +1,20 @@
+# Contrast Maven Plugin
+
+The Contrast Maven Plugin helps users include one or more Contrast Security analysis features in
+their Java web application Maven projects.
+
+## Goals Overview
+
+* [contrast:install](install-mojo.html) includes the Contrast Java agent in integration testing to
+  provide Contrast Assess runtime security analysis
+* [contrast:verify](verify-mojo.html) verifies that none of the vulnerabilities found by Contrast
+  Assess during integration testing violate the project's security policy (fails the build when
+  violations are detected)
+* [contrast:scan](scan-mojo.html) analyzes the Maven project's artifact with Contrast Scan to find
+  vulnerabilities using static analysis
```

Comment:
Is there a reason we don’t end these statements with punctuation?

Reply:
Good question. I'm always asking myself "does this bullet point need to end in punctuation?" and until you asked I never bothered to look up the right answer. The [Contrast Style Guide](https://contrast.atlassian.net/wiki/spaces/DT/pages/2214953860/Contrast+Security+Language+and+Style+Guide) doesn't directly address this but it defers to [Microsoft Style Guide](https://docs.microsoft.com/en-us/style-guide/welcome/) for everything it doesn't directly address. The Microsoft Style Guide [would use ending punctuation here](https://docs.microsoft.com/en-us/style-guide/punctuation/periods), because these bullets are complete sentences.

I'll fix it, and I'm happy to have finally figured out what the rules are 😄